src/components/molecules/MultiStepForm/Steps/Step.styled.ts

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+
+export const StyledWrapper = styled.div`
+  display: flex;
```

Comment:
if we have the same styling in `StyledWrapper` and `StyledContainer` could we merge it into the same style component?